setup.py

```diff
@@ -57,6 +67,26 @@ def is_requirement(line):
     )
 
 
+def get_edx_platform_version():
+    """
+    Returns the pip package version for Open-edX
+    """
+    file_name = 'open_edx_version.txt'
+    command = 'pip show Open-edX > {}'.format(file_name)
+    subprocess.call(command, shell=True)
+    version = 0.0
+
+    with open(file_name, 'r') as file:
+        lines = file.readlines()
```

Comment:
Please, add a break line here to separate the assignation statements and other Python statements. @andrey-canon